compiler/test/graindoc/types.expected.md

```diff
@@ -0,0 +1,52 @@
+---
+title: TypeGrainDoc
+---
+
+## Types
+
+Type declarations included in the TypeGrainDoc module.
+
+### TypeGrainDoc.**R**
+
+```grain
+record R {
+  x: Number,
+  y: String,
+}
+```
+
+A record
+
+Fields:
+
+|field name|description|
+|----------|-----------|
+|`x`|Record field x|
+|`y`|Record field y<br />Second line|
+
+### TypeGrainDoc.**E**
+
+```grain
+enum E {
+  Variant1,
+  Variant2(String, Number),
+}
+```
+
+An enum
+
+Variants:
+
+|variant name|description|
+|------------|-----------|
+|`Variant1`|Enum variant|
+|`Variant2`|Another variant|
```

Comment:
I question if this should contain the type info in here as well like 
```suggestion
|`Variant2`|(String, Number)|Another variant|
```
Similar to the docs for functions

Reply:
Sounds like a good idea to have a separate type column for records, although for variants I'm not sure that this is the best way to display this info, perhaps it would be better to just display the whole variant description in the first column like `Variant2(String, Number)`?

Reply:
Maybe, I think that it is worth thinking about consistency if we have functions with a separate type column and use a separate type column for records we should probably use a separate column for variants. I think that things might get a little messy with records attached to variants though so this might need some discussion or be something we come back to in another pr later on.